bonsai-core/src/test/scala/com/stripe/bonsai/BitsetSpec.scala

```diff
@@ -18,7 +18,7 @@ class IndexedBitSetSpec extends WordSpec with Matchers with Checkers {
 
   implicit val arbitraryBitSetWithIndex: Arbitrary[BitSetWithIndex] =
     Arbitrary(for {
-      len <- arbitrary[Short].map(_.toInt)
+      len <- arbitrary[Short].map(_.toInt) if len >= 0
```

Comment:
ahh, yeah. We are resulting in an `Int`.

So, in that case, I don't like `_ & 0xffff` because all negative numbers are mapped to `65535`, so half the time we might see a really big list. I think we are better off with:
`x => if(x < 0) -(x.toInt) else x`